activerecord/CHANGELOG.md

```diff
@@ -31,6 +31,13 @@
 
     *Prathamesh Sonpatki*
 
+*   Support index length and order options using both string and symbol
+    column names.
+
+    Fixes #27243.
+
+    *Ryuta Kamizono*
+
```

Comment:
Yes because it doesn't get added in order of commits but rather at the top of the file to avoid conflicts.